ecs.php

```diff
@@ -30,4 +30,8 @@
         __DIR__ . '/node_modules',
         __DIR__ . '/storage',
     ]);
+
+    $parameters->set(Option::SKIP, [
+        PhpCsFixer\Fixer\PhpUnit\PhpUnitStrictFixer::class,
```

Comment:
Anders wil hij de `assertEquals` in mijn nieuwe test vervangen door `assertSame` maar die werkt niet voor wat ik test, en ik weet niet of het mogelijk is om binnen een file een fixer te skippen.

Reply:
Check, klopt, super hinderlijk.